tests/test-execute-callback.cpp

```
@@ -0,0 +1,159 @@
+    desc.callback = executeCallback;
+    desc.userObject = record;
+    desc.retainUserObject = retainCallbackObject;
+    desc.releaseUserObject = releaseCallbackObject;
+    desc.userData = &payload;
+    desc.userDataSize = sizeof(payload);
+    encoder->executeCallback(desc);
+}
+
+} // namespace
+
+GPU_TEST_CASE("execute-callback-native-handle-and-user-data", D3D12 | Vulkan | CUDA)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DOCTEST_GLOBAL_NO_WARNINGS is a macro then please configure it.

(unknownMacro)